libpod/options.go

```diff
@@ -134,9 +134,9 @@ func WithConmonPath(path string) RuntimeOption {
 		if rt.valid {
 			return ErrRuntimeFinalized
 		}
-
-		rt.config.ConmonPath = path
-
+		if path != "" {
```

Comment:
We should make "" throw an error once the libkpod config is entirely gone, and we only pass this when it's actually needed. Can you throw a TODO on here about that?